### DISCUSSION TITLE
Public importer gives me "500 SERVER ERROR"

### DISCUSSION THREAD

**Discussion by ementat** [firefly-iii/firefly-iii#8055] [General]
Hi!

I am trying to use public importer tool https://data-importer.firefly-iii.org/. Connection to my Firefly III instance went well, but every time I am trying to upload my csv file all I get is a "500 SERVER ERROR". Any suggestions what can be done/checked in this situation?

Dmitry

**Comment by JC5** [comment]
I can check the logs, thanks for letting me know!

**Comment by JC5** [comment]
If you try again, let me know.

**Reply by ementat** [reply]
Great! It is working now. I reached the configuration screen :) Thanks!

**Comment by ementat** [comment]
Conversion process failed with an error: "The conversion could not be started, or failed due to an error. Please check the log files. Sorry about this :(".

**Reply by JC5** [reply]
Mmmm, I see a few duplicate transaction errors but no real failures.

**Reply by ementat** [reply]
<img width="601" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/4093563/314aa393-309e-416e-91d4-1dc3b86484af">

**Reply by JC5** [reply]
I think I found it:

```
[2023-10-13 10:11:16] local.ERROR: Maximum execution time of 150 seconds exceeded {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Maximum execution time of 150 seconds exceeded at /var/www/data-importer/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:595)
```

That's not something I can help, I'm afraid. It looks like it just took to long to connect to your server and import the transactions.
